pallets/msa/src/runtime-api/src/lib.rs

```diff
@@ -31,7 +31,7 @@ sp_api::decl_runtime_apis! {
 		// fn get_msa_keys(msa_id: MessageSourceId) ->	Vec<KeyInfoResponse<AccountId>>;
 
 		/// Check to see if a delegation existed between the given delegator and provider at a given block
-		fn has_delegation(delegator: Delegator, provider: Provider, block_number: Option<BlockNumber>) -> bool;
+		fn has_delegation(delegator: Delegator, provider: Provider, block_number: BlockNumber, schema_id: Option<SchemaId>) -> bool;
```

Comment:
This needs to be reflected in `MsaRuntimeApi` under `js/api-augment/definitions/msa.ts` as well.

Reply:
Strange.  I do not see this in my code.  My code has:

`fn has_delegation(delegator: Delegator, provider: Provider, block_number: BlockNumber, schema_id: Option<SchemaId>) -> bool;`